src/main/java/seedu/exception/InvalidTelegramUsernameException.java

```diff
@@ -0,0 +1,4 @@
+package seedu.exception;
+
+public class InvalidTelegramUsernameException extends Exception{
```

Comment:
If I am not wrong, there should be a space before the curly brace, `Exception {`